src/helpers.ts

```diff
@@ -1,3 +1,5 @@
 export const logger = data => {
+  console.log("logs:start");
```

Comment:
/nit logs should contain meaningful structured data. Would recommend using pino for logging

Reply:
done